Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt-get update && apt-get install -y \
 	&& apt-get clean \
 	&& rm -rf /var/lib/apt/lists/*
 
-RUN pip install --upgrade pip && pip install pytest lxml cssselect pillow==4.1.0
+RUN pip -vv install --upgrade pip==9.0.1 && pip -vv install pytest==3.6.0 lxml cssselect pillow==4.1.0
```

Comment:
What is `-vv`? Is it needed?

Reply:
I think this part is typo. I just want to show verbose mode to check errors, but actual arg are -v or --verbose. But it seems working and show verbose logs I will change this part to --verbose to help reviewers.

Reply:
Or you can remove `--verbose` for now, since people usually won't need detailed debug log of pip (when it works). We can add it back later for analysis when needed again. It's up to you though.